stratagem-runner/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "stratagem-runner"
+version = "0.1.0"
+authors = ["IML Team <iml@whamcloud.com>"]
+edition = "2018"
+
+[dependencies]
+liblustreapi = { path = "liblustreapi", version = "0.1.0" }
+
+[[bin]]
+name = "stratagem_warning"
+path = "src/stratagem_warning.rs"
+
+[[bin]]
+name = "stratagem_purge"
+path = "src/stratagem_purge.rs"
+
+
+[workspace]
+members = ['liblustreapi', 'liblustreapi-sys']
```

Comment:
These can't be nested, move these members up to the workspace root at: integrated-manager-for-lustre